help/CreateImageAndAzureResources.md

```diff
@@ -124,3 +124,34 @@ Generated tool versions and details can be found in related projects:
 - [Go](https://github.com/actions/go-versions)
 - [Node](https://github.com/actions/node-versions)
 - [Boost](https://github.com/actions/boost-versions)
+
+### Post-generation scripts
+Extra configuration should be applied after packer build process, because packer build user and default agent's user are not the same. These configuration scripts are located in the `post-generation` folder. 
+
+The scripts are copied to the virtual machines during the packer build to `/opt/post-generation` directory for Ubuntu and to `C:\post-generation` directory for Windows images.
+
+Use the following commands to run these scripts::
+
+- **Ubuntu**
+
+        for i in $(find /opt/post-generation -mindepth 1 -maxdepth 1 -type f -name '*.sh'); do bash "$script"; done
```

Comment:
You can use find --exec instead of for loop.
https://www.baeldung.com/linux/find-exec-command